apps/microsoft-teams/frontend/src/components/config/AccessSection/AccessSection.tsx

```diff
@@ -39,10 +44,56 @@ const AccessSection = (props: Props) => {
   const { login, teamsAppInfo, teamsAppLink, description, authError, orgDetailsError } =
     accessSection;
 
+  // saves the app config page after dispatch completes and parameters are updated
+  const saveMsAccountInfo = async () => {
+    await customApi.saveConfiguration();
+    setHasUpdatedMsAuth(false);
+    setHasUpdatedOrgDetails(false);
+    setHasDisconnected(false);
+    setUpdatedMsAccountInfo(parameters);
+  };
+
+  useEffect(() => {
```

Comment:
My intention here was to ensure that we were calling `saveConfiguration` _after_ the new parameter data had been dispatched. This is because `saveConfiguration` uses what's in parameters to save, rather than us passing in the new parameters to the `saveConfiguration` call. Definitely open to feedback on this if there is a better way!